content/post/2024/02-sky130-kicad.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "sky130 SPICE, the KiCad way"
+date: 2024-02-12T19:09:55-0800
+description : "SPICE your ASIC schematics with KiCad!"
+tags: ["blog"]
+image: /static-2024/sky130-kicad/banner.png
+slug: sky130-kicad-spice
+draft: false
+---
+
+_This guest post is by [Aki Van Ness]._
+
+# sky130 SPICE, the KiCad way
+
+Picture this, you're working on a custom layout for [sky130], be it a PLL, or you're just trying your hands at making some basic building blocks. You've got a plan, and and ready to start laying down some nets, and that's when you must face it, the dreaded [xschem]!
+
+All joking aside, xschem is an incredibly powerful schematic capture system, but that power comes at a cost. xschem is fairly arcane to most, it has some graphical problems, and is just in general really hard to use if you're just starting out which makes rapid iteration difficult.
+
+Enter [KiCad], a robust and Open Source EDA software package, while it is mainly used for PCB layout, we can leverage the schematic capture and built-in SPICE simulation, right?
+
+## Introducing kicad-pdk-libs
+
+A little while ago I ran into [that exact problem](https://twitter.com/lethalbit/status/1498345181359321091), and so I set out and made [kicad-pdk-libs], a KiCad symbol library that adds schematic symbols for not only the basic building blocks of the sky130 PDK, but also SPICE library linkage.
+
+With this you can do things as simple as just stubbing out a SPICE simulation for some of the primitive gates, or go down to the transistor level and build gates from scratch and simulate their characteristics!
+
+![The KiCad "Choose Symbol" dialog showing a symbol called "sky130_nfet_01v8" being selected](/static-2024/sky130-kicad/kicad_nfet.png)
+
+As it stands, [kicad-pdk-libs] has almost 100% complete symbols for both sky130A and sky130B, for all the cell libraries in addition to the base `sky130_fd_pr` primitives library. This allows you to not just put transistor level schematics together, but much larger schematics using all of the pre-made gates and cells in the library, including some SRAM blocks.
```

Comment:
suggestion for rewording the first sentence:  for both the Sky130A and Sky130B PDKs, all the cell libraries as well as the base ...